src/util/i18nFormatting.js

```diff
@@ -0,0 +1,30 @@
+function getVariationOfAOrAn(value) {
+  const vowelishLetters = ['a', 'e', 'i', 'o', 'u', 'h'];
+  const isVowelish = vowelishLetters.find(l => value.substring(0, 1) === l);
+  return isVowelish ? 'an' : 'a';
+}
+
+/*
+Arguments:
+value:   The value of the variable that is being passed
+  in to be interpolated
+format:  the pipe-delimited list of formatting flags,
+  it will run the formatting functions sequentially
+  based on the sequence of the flags passed in.
+lng: the language used for the formatting
+NOTE: lng is available(from i18Next in this function) but is not
+currently used, so I've removed it from the arguments list.
+*/
```

Comment:
I’m not philosophically opposed to API doc style comments like this, but I think we should probably use the [JS Doc](http://usejsdoc.org/about-getting-started.html) format?